typescript/src/openapi-client/cdpApiClient.ts

```diff
@@ -115,6 +121,16 @@ export const cdpApiClient = async <T>(
     const response = await axiosInstance(configWithIdempotencyKey);
     return response.data as T;
   } catch (error) {
+    // eslint-disable-next-line import/no-named-as-default-member
+    if (Axios.isAxiosError(error) && !error.response) {
+      throw new UnknownApiError(
+        HttpErrorType.unknown,
+        error.cause instanceof Error ? error.cause.message : error.message,
+        `${ERROR_DOCS_PAGE_URL}#unknown`,
```

Comment:
If keeping this, this docs page would have to be created

Reply:
Let's not introduce links to pages that don't exist yet